src/components/Navigation.jsx

```diff
@@ -1,12 +1,16 @@
 import { Link } from "react-router-dom"
+import '../App.css'
 
 const Nav = () => {
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to follow the provided design, the profile icon should be in blue and the yellow line at the bottom of the nav bar should not be present in your app

![Screenshot_20230520_155742](https://github.com/OnlyUnknown/book-store/assets/25227022/f9bcbc98-57e7-4fff-9f23-42a8275c5b36)